Flow.Launcher/SettingPages/Views/SettingsPaneTheme.xaml

```diff
@@ -112,37 +112,39 @@
                                     Visibility="Visible" />
 
                                 <flowlauncher:ResultListBox
+                                    x:Name="ResultListBox"
                                     Grid.Row="2"
                                     DataContext="{Binding PreviewResults, Mode=OneTime}"
                                     Visibility="Visible" />
-
+                                <Border x:Name="ContextMenu" Visibility="Collapsed" />
```

Comment:
There is a problem of attempting to control the context menu and history that do not exist in the preview area. So I added here.

Reply:
It is always Collapsed, then why would it be needed?

Reply:
The preview area is created by importing the mainwindow. There is code in the main window code that tries to draw the contextmenu and history, which works here as well. We didn't insert that part here at all, because the visibility behavior code tries to touch the contextmenu and history, which causes a binding error. So I added an invisible layer with the same name to receive visibility related behaviors. Purely for error prevention.